app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    people_list = []
+    for person in people:
+        people_list.append(Person(person["name"], person["age"]))
```

Comment:
use get() method instead of parenthesis everywhere to avoid KeyError

Reply:
<img width="1440" alt="problem" src="https://user-images.githubusercontent.com/67129371/219868630-aee427af-3d43-45ee-9a6a-fe3c140c739a.png">
Here I am using get() everywhere I can, but still getting KeyError. Could you, please, give any suggestions?

Reply:
@Bohdan-Salamakha No need .get here